src/nonce.iced

```diff
@@ -0,0 +1,18 @@
+nonceBytes = 24
+sigNonceBytes = 16
+{prng} = require('crypto')
+uint64be = require('uint64be')
+
+exports.nonceForSenderKeySecretBox = () -> return new Buffer('saltpack_sender_key_sbox')
+
+exports.nonceForPayloadKeyBox = () -> return new Buffer('saltpack_payload_key_box')
+
+exports.nonceForMACKeyBox = (headerHash) ->
+  if headerHash.length isnt 64 then return new Error('Header hash shorter than expected')
```

Comment:
It feels a little weird to return a single value that can either be an error or a buffer. Since this condition is probably a programming bug, maybe it makes more sense to throw?